### ISSUE TITLE
[bug] Calling bottle :unneeded is deprecated! There is no replacement.

### ISSUE THREAD

**Issue by klonos** [fubarhouse/pygmy#344]
**Describe the bug**
When running `brew install pygmy-go` after having run `brew tap fubarhouse/pygmy-go`, I get a series of these warnings:

```
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the fubarhouse/pygmy-go tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/fubarhouse/homebrew-pygmy-go/Formula/pygmy-go.rb:9

==> Downloading https://github.com/fubarhouse/pygmy-go/releases/download/v0.7.1/pygmy-go_0.7.1_darwin_amd64.tar.gz
==> Downloading from https://github-releases.githubusercontent.com/215281006/db920b6e-adf6-423e-a0a4-80d303aa8a91?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz
######################################################################## 100.0%
==> Installing pygmy-go from fubarhouse/pygmy-go
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the fubarhouse/pygmy-go tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/fubarhouse/homebrew-pygmy-go/Formula/pygmy-go.rb:9

Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the fubarhouse/pygmy-go tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/fubarhouse/homebrew-pygmy-go/Formula/pygmy-go.rb:9

🍺  /usr/local/Cellar/pygmy-go/0.7.1: 5 files, 9.6MB, built in 10 seconds
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the amazeeio/lagoon-cli tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/amazeeio/homebrew-lagoon-cli/lagoon.rb:8
```

**Comment by fubarhouse** [comment]
Thanks for that Greg - I'll check this out after hours. :+1:

**Comment by tobybellwood** [comment]
it's been fixed in goreleaser https://github.com/goreleaser/goreleaser/releases/tag/v0.183.0 - but that does also require a little re-work to binary packaging - I might see if I can get a handle on it.

**Comment by tobybellwood** [comment]
Fixed in the 0.8.0 release https://github.com/fubarhouse/pygmy-go/releases/tag/v0.8.0

**Comment by klonos** [comment]
Thanks 🙏🏼 ...confirming that I don't see these warnings any longer 👍🏼
